components/google-cloud/aiplatform/sdk/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is the additional nesting under aiplatform/sdk needed? Would recommend moving it back out like you had before.

Reply:
Usually the users come to the `components/*` directory to search for components.
For the users, the sdk code is more like an implementation detail. It makes sense to keep everything related to the sdk package in one place/directory, without interspersing it with other entities like MB components.
We can put the actual MB components front and center inside `components/google-cloud/aiplatform/`:
* aiplatform/Dataset/Create
* aiplatform/Dataset/Import_data
* aiplatform/Dataset/Export_data
* aiplatform/Model/Train
* aiplatform/Model/Deploy
* aiplatform/Model/Export
etc

Reply:
This is planned as an SDK-only components package so this indirection is unnecessary. Also, it's planned to host other GCP components, not just for ai platform.

Reply:
Acknowledged, I will clean it up in a follow up PR tracked under ( b/184538663). Reasoning to do it in a follow up PR is that I want to make sure we make progress on the other fronts (samples, testing, etc). Since this does not change the user experience with sdk code directly we can change the structure fairly easily as needed.